.github/CONTRIBUTING.md

```diff
@@ -31,6 +31,7 @@ commands are available:
   run as a git `pre-commit` hook.
 - `yarn build` to rebuild the Gatsby site JS, CSS, and HTML. The build runs
   as part of the initial install.
+- `yarn run upgrade` - upgrades the `react-component` git submodule and component packages
```

Comment:
```suggestion
- `yarn run upgrade` to upgrade Garden component packages and sync the `react-components` submodule.
```

- grammar alignment with existing bullets (incl. period)
- correction re: `react-components`
- written to correspond with script order-of-operation